### ISSUE TITLE
TransformVlcBitmap: Why we need to convert the bitmap from 160x480 to 240x320

### ISSUE THREAD

**Issue by TooSchoolForCool** [Microsoft/HoloLensForCV#81]
In the file [FrameRenderer.cpp](https://github.com/Microsoft/HoloLensForCV/blob/master/Samples/SensorStreamViewer/FrameRenderer.cpp). What is the function TransformVlcBitmap() going to do?

I believe the image stream from the VisibleLight cameras is 160x480. However it the function TransformVlcBitmap() it tries to store such 160x480 bitmap into a 240x320. Why we need to do this?

**Comment by yogihamdani** [comment]
My guess from several raw pictures that I took, is that in 160x480 the image is stretched. Meanwhile with 240x320, the image is at its correct scale. Though, I am not completely sure about this. I am also looking for a proper specification and details regarding the environment understanding sensor camera.

**Comment by TooSchoolForCool** [comment]
Right. Literally, there are not many official documentations for HoloLens research mode...
